.github/workflows/cd-docker.yml

```diff
@@ -0,0 +1,20 @@
+name: SMARTS CD Docker
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  docker:
+    name: Build and push Docker image to Docker Hub
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v1
```

Comment:
what is `docker/build-push-action`? is this a script we wrote or some built-in script